new/LIT6276MGChurchkeeper.xml

```diff
@@ -0,0 +1,108 @@
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="SH" when="2020-12-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusText"/>
+                        <citedRange unit="page">31–33 (30th miracle)</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusTrans"/>
+                        <citedRange unit="page">18–19 (30th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:Strelcyn1974CatJRylands"/>
+                        <citedRange unit="page">36</citedRange>
+                        <citedRange unit="item">Ms 12, II.31 (30th miracle)</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:GrebTiss1935Codices"/>
+                        <citedRange unit="page">723</citedRange>
+                        <citedRange unit="item">Vat 233, 3.30 (30th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listRelation>
+                    <relation name="saws:formsPartOf" active="LIT6276MGChurchkeeper"
+                        passive="LIT4006Taamme"/>
+                </listRelation>
+
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="BDLaethb4"/>.</note>
+                    <ab>፴ ተአምሪሁ፡ <gap reason="ellipsis" resp="SH"/>ወሀለወት፡ አሐቲ፡ ቤተ፡ ክርስቲያን፡ ሐዳስ፡ እንተ፡ ሐነጸዋ፡ በስመ፡
+                        ቅዱስ፡ ማር፡ ጊዮርጊስ፡ ውስተ፡ መካን፡ ዘነበረ፡ ማሪት።
+                        ወበእመዋ<supplied reason="omitted" resp="SH">ዕ</supplied>ል፡
```

Comment:
Perhaps here I would also add `<supplied reason="omitted" resp="SH">፩</supplied>` after በ because the two prepositions በ- እም- are rarely found together (like e.g. እምውስተ, እምታሕተ, በኀበ etc. ...), so something seems necessary in the middle.

Reply:
I forgot ፩! It is in the text, I corrected it.